hbase-common/src/main/java/org/apache/hadoop/hbase/io/ByteBuffAllocator.java

```diff
@@ -92,20 +92,6 @@
   @Deprecated
   static final String DEPRECATED_BUFFER_SIZE_KEY = "hbase.ipc.server.reservoir.initial.buffer.size";
 
-  /**
-   * The hbase.ipc.server.reservoir.initial.max and hbase.ipc.server.reservoir.initial.buffer.size
-   * were introduced in HBase2.0.0, while in HBase3.0.0 the two config keys will be replaced by
-   * {@link ByteBuffAllocator#MAX_BUFFER_COUNT_KEY} and {@link ByteBuffAllocator#BUFFER_SIZE_KEY}.
-   * Also the hbase.ipc.server.reservoir.enabled will be replaced by
-   * hbase.server.allocator.pool.enabled. Keep the three old config keys here for HBase2.x
-   * compatibility.
-   */
```

Comment:
The original author made an effort to document that this deprecation happened as part of v2.0.0, so please preserve this in a comment adjacent to the config's `DeprecationDelta`.